src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -501,6 +501,13 @@ def load_arguments(self, _):
         c.argument('protect_from_scale_in', arg_type=protection_policy_type, help="Protect the VM instance from scale-in operations.")
         c.argument('protect_from_scale_set_actions', arg_type=protection_policy_type, help="Protect the VM instance from scale set actions (including scale-in).")
 
+    for scope in ['vmss create', 'vmss update']:
+        with self.argument_context(scope) as c:
+            c.argument('terminate_notification', min_api='2019-03-01', arg_type=get_three_state_flag(),
```

Comment:
Usually we use `enable_terminate_notification`

Reply:
https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-create

`vm create` has 3 boolean parameters now. They don't have a enable_ prefix.

`enable_` means you can only enable it. You can not pass true or false.

Reply:
Abbreviation is not urgent. If this parameter is used heavily, we can add it.

Reply:
`three_flags_arguments` supports user just inputs `--terminate-notification` without any value followed. So it's quite confusing to use this argument like that.

Reply:
I followed the pattern here.
https://github.com/Azure/azure-cli/blob/844005a08d19ab7a5544aa9147869e101426ea67/src/azure-cli/azure/cli/command_modules/vm/_params.py#L495
https://docs.microsoft.com/en-us/cli/azure/vmss?view=azure-cli-latest#az-vmss-create

Reply:
https://docs.microsoft.com/en-us/cli/azure/vmss?view=azure-cli-latest#az-vmss-create
Observation: parameters in same logic group share same prefix.

Reply:
Well, I think `enable_accelerated_networking` will be better. I discussed with @zikalino about it before. Taking this chance, we can discuss it again and follow the rule in the future.

Reply:
@zikalino What's your thoughts about `enable_` and `get_three_state_flag()`? We can make a conclusion and  I will update the [command authoring doc](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

Reply:
It's time to escalate.
@yugangw-msft What's the best practice?

Reply:
Just in suggestion, for storage module, we also use `enable_xxx` for such argument to enable a flag or argument.

Reply:
There is one parameter called "--ultra-ssd-enabled" in below doc:
https://docs.microsoft.com/en-us/cli/azure/vmss?view=azure-cli-latest#az-vmss-create
Maybe you can also follow that pattern and use "terminate-notification-enabled" so that these two parameters can share the same prefix "terminate-notification".
@yugangw-msft can make the call for name convention.

Reply:
let us go with `enable_terminate_notification`. `accelerated_networking` is a questionable naming which I probably should not have named that way.

Reply:
Thank you for your reviews.